Automattic/wp-calypso
# Reader: Add new gallery block styles and move rendered block styles to own file

#### Changes proposed in this Pull Request

* Add additional styles to allow the Reader to render the refactored gallery block
* Move the rendered block styles mixins to their own file as typography doesn't seem like a good fit

#### Testing instructions

* Check out the PR and in a site with `enable-gallery-block-refactor` blog sticker
* Add a gallery block with a number of images
* View the post in Reader and make sure it is rendered in a grid and not as a single column

Before:

<img width="631" alt="Screen Shot 2021-10-07 at 3 24 55 PM" src="https://user-images.githubusercontent.com/3629020/136310454-a806adba-b5cc-47a6-9e06-252032daf9b5.png">

After:

<img width="775" alt="Screen Shot 2021-10-07 at 3 11 53 PM" src="https://user-images.githubusercontent.com/3629020/136310369-41bc8620-b83c-43ae-bb93-d2723c311509.png">

Related to #56307
